package.json

```diff
@@ -114,7 +114,7 @@
     "path-browserify": "^1.0.1",
     "pixelmatch": "5.3.0",
     "postcss": "^8.2.8",
-    "prettier": "2.8.3",
+    "prettier": "2.8.4",
```

Comment:
Not sure how this change found it's way into this PR 🤔

Reply:
Probably a case of v3.1-feature being created before we updated Prettier, but this feature branch being created with it?

Reply:
But I agree, we should rm this (likely by rebasing the 3.1 feature branch on `main`, then this off the 3.1 feature branch)

Reply:
oops haha

I should inspect the diff before I open the PR!